packages/augur-sdk/src/state/db/MarketDB.ts

```diff
@@ -85,57 +85,55 @@ export class MarketDB extends DerivedDB {
     }
   }
 
-  syncOrderBooks = async (syncing: boolean): Promise<void> => {
+  syncOrderBooks = async (marketIds: string[]): Promise<void> => {;
```

Comment:
I think this still needs some sort of debouncing behavior on it such that we only recalc liquidity on some fixed cycle (could be as long as once per 5 sec or something)

Reply:
Imo should only recalculate once a minute or so, doesn’t need to be super frequent